styles/styles.css

```diff
@@ -305,6 +305,11 @@ main img {
   main .section > div {
     width: 1176px;
   }
+
+  /* section only visible on mobile */
+  main .section.desktop-hidden {
```

Comment:
Is this change related, or just something extra that was missing?

Reply:
That is related we have special sections aka section metadata with this behavior.